### PR TITLE
Upgrade to openRTB 2.6: add changes made to old fork

### PR DESCRIPTION
**Pull request by Araknyfe**

### CODE REVIEW

File: openrtb2/bid.go [hunk 1]
```diff
@@ -104,6 +105,23 @@ type Bid struct {
...
 	AdM string `json:"adm,omitempty"`
 
+	// Attribute:
+	//   adm_native
+	// Type:
+	//   object
+	// Description:
+	//   Optional response for BidSwitch Native
+	//	 https://protocol.bidswitch.com/ssp-protocol/ssp-response-native-object.html
+	AdMNative native1.Response `json:"adm_native,omitempty"`
```
Comment (SyntaxNode) [inline openrtb2/bid.go:115]: These are not OpenRTB 2.6 fields. This package implements the [official standard](https://github.com/InteractiveAdvertisingBureau/openrtb2.x/blob/main/2.6.md) and does not accept vendor specific changes.

File: adcom1/app.go [hunk 2]
```diff
@@ -80,6 +80,10 @@ type App struct {
 	//   string
 	// Definition:
 	//   Bundle or package name of the app (e.g., “com.foo.mygame”) and should NOT be app store IDs (e.g., not iTunes store IDs).
+	//   A platform-specific application identifier intended to be
+	//   unique to the app and independent of the exchange. On
+	//   Android, this should be a bundle or package name (e.g.,
+	//   com.foo.mygame). On iOS, it is typically a numeric ID.
```
Comment (SyntaxNode) [inline adcom1/app.go:86]: The existing description matches the [official AdCom specification](https://github.com/InteractiveAdvertisingBureau/AdCOM/blob/main/AdCOM%20v1.0%20FINAL.md#object_app). If you seek to change it, please open an issue with the IAB.

File: adcom1/linearity_mode.go [hunk 3]
```diff
@@ -1,6 +1,12 @@
 package adcom1
 
 // LinearityMode represents options for media linearity, typically for video.
+// Options for video linearity.
+// “In-stream” or “linear” video refers to preroll, post-roll, or mid-roll video ads where the user is forced to watch ad in order to see the video content.
+// “Overlay” or “non-linear” refer to ads that are shown on top of the video content.
+//
+// This OpenRTB list has values derived from the Inventory Quality Guidelines (IQG).
+// Practitioners should keep in sync with updates to the IQG values.
```
Comment (SyntaxNode) [inline adcom1/linearity_mode.go:9]: The list values is from the [official AdCom specification](https://github.com/InteractiveAdvertisingBureau/AdCOM/blob/main/AdCOM%20v1.0%20FINAL.md#list_linearitymodes).

**Review by SyntaxNode — COMMENTED** [review]
The changes proposed in this PR do not follow the specifications from the IAB.

This project was originally a fork of https://github.com/mxmCherry/openrtb designed for Prebid Server, but was abandoned (in 2018?) when Prebid Server switched back to using the official https://github.com/mxmCherry/openrtb repository.

In October 2022, this project was re-synced with the https://github.com/mxmCherry/openrtb origin and became the official continuation of that project.

Is this the "old fork" you refer to?

File: openrtb2/bid.go (hunk 1 above)
Comment (Araknyfe) [reply openrtb2/bid.go:115]: Sorry this was a mishap on our pulls, thanks for taking the time!

**Pull request closed without merging**